reference/core/ingress-controller.md

```diff
@@ -140,7 +140,7 @@ This means that the following YAML:
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.